core/src/main/java/org/hisp/dhis/android/network/attribute/AttributeValueDTO.kt

```diff
@@ -31,15 +31,16 @@ package org.hisp.dhis.android.network.attribute
 import kotlinx.serialization.Serializable
 import org.hisp.dhis.android.core.attribute.AttributeValue
 import org.hisp.dhis.android.network.common.dto.ObjectWithUidDTO
+import org.hisp.dhis.android.network.common.dto.ValueDTO
 
 @Serializable
 internal data class AttributeValueDTO(
-    val value: String?,
+    val value: ValueDTO?,
```

Comment:
It does not hurt, but this is not a data value that needs to post a null. This is only for metadata attribute values